### PR TITLE
Power of Two improvements. avoid loop and recursion.

### PR DESCRIPTION
**Pull request by alexpantyukhin**
#### Description of Change

Add a guide for new LeetCode solutions.

#### Checklist

- [x] Added description of change
- [x] Added file name matches [File name guidelines](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md#File-Name-guidelines)
- [ ] Added tests and example, test must pass
- [x] Relevant documentation/comments is changed or added
- [x] PR title follows semantic [commit guidelines](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md#Commit-Guidelines)
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.

Notes:

### CODE REVIEW

File: leetcode/src/231.c [hunk 1]
```diff
@@ -1,7 +1,6 @@
...
-        return false;
-    while (n % 2 == 0) n /= 2;
-    return n == 1;
-}
\ No newline at end of file
+// Without loops/recursion.
+// Runtime: O(1)
+// Space: O(1)
+bool isPowerOfTwo(int n){
+    return (n > 0) && ((n & (n - 1)) == 0);
```
Comment (Panquesito7) [inline leetcode/src/231.c:5]: Shouldn't this be `&&`?
```suggestion
    return (n > 0) && ((n && (n - 1)) == 0);
```

Comment (alexpantyukhin) [reply leetcode/src/231.c:5]: No, actually this is a bitwise and for 2 integers https://en.wikipedia.org/wiki/Bitwise_operations_in_C#Bitwise_AND_& .

**Review by Panquesito7 — APPROVED** [review]
Awesome. Tested and works. Thanks! 🚀

**Review by tjgurwara99 — APPROVED** [review]
Nice use of bitwise operations!

**Pull request merged**